src/main.py

```diff
@@ -0,0 +1,42 @@
+from leader import Leader
+from follower import Follower
+from tagrec import TagRecognition
+from inputcontroller import InputController
+
+
+"""
+Main script to run follower and leader functionality
+
+Takes input from camera to determine if current vehicle 
+should lead or follow based on detecting an AR tag.
+
+If an AR tag exists the follower functionality will run.
+If an AR tag does not exist the leader functionality will run.
+"""
+def main():
+
+    tag = TagRecognition()
+
+    if tag.detect() is not None:
+        follower = Follower()
+        while True:
+            follower.follow()
+
+    else:
+        leader = Leader()
+        controller = InputController()
```

Comment:
I don't really like how this is essentially leader.main, but there isn't really much more it should be doing. I'm going to wrap this in a function similar to follower.follow(). Expect that pull request in a few minutes and adjust this accordingly please.